docs/README.md

```diff
@@ -71,3 +71,4 @@ covered by the section above.
 | Vkontakte | _Optional_ | [Documentation](vkontakte.md "Vkontakte provider documentation") |
 | Weibo | _Optional_ | [Documentation](weibo.md "Weibo provider documentation") |
 | WorkWeixin (WeCom) | _Optional_ | [Documentation](workweixin.md "WorkWeixin provider documentation") |
+| Xero | _Optional_ | [Documentation](xero.md "Xero provider documentation") |
```

Comment:
Can you also add a row to table of providers in the main README please? There's a template in a comment at the end of the table.